Update components for GEFS pre-production parallel

WalterKolczynski-NOAA opened the pull request:
The version of GSI and post are updated to new tags that reflect
changes made for the official NCO parallel. The repository for GSI
is also changed to GitHub to reflect the new official repository.

WalterKolczynski-NOAA committed:
Update GSI and post for GEFS official parallel

The version of GSI and post are updated to new tags that reflect
changes made for the official NCO parallel. The repository for GSI
is also changed to GitHub to reflect the new official repository.

XianwuXue-NOAA reviewed (commented):
Walter, you changed the GSI from slab to GitHub, however, I think we should ask Xiaoxue/NCO to see whether this is good because they didn't test the GitHub version.

WalterKolczynski-NOAA commented:
There are already several other components that use GitHub. It shouldn't be a problem.

XianwuXue-NOAA commented:
Yes, please check these two gsi(vlab, GitHub) are identical. I just talked with Xiaoxue, she said, if they are identical, it is OK, otherwise, they must do the extra test and postpone the parallel test.

WalterKolczynski-NOAA commented:
Three tarballs were removed that are presumably in the fix directory now, but it's not something we use anyway because we don't do any data assimilation.

XianwuXue-NOAA commented:
No problem for me. I am afraid that NCO don't think so. Like the the symbolic links in ufs_util, and we don't use them, however, they asked us to remove it.

WalterKolczynski-NOAA committed:
Move utils linking out of build script

The linker script for ufs_utils was being called from the build
script instead of the master link script and was hard-coded to
use emc (symlink). The call is now moved to the master linker
script and properly uses the command line argument to determine
whether to copy (nco) or use symlinks (emc).

WalterKolczynski-NOAA committed:
Update prep_chem version for pre-production parallel

One of the script in prep_chem had to be renamed to meet NCO
guidelines.

XianwuXue-NOAA commented:
I just compared the gsi_vlab vs gsi_github, and below are the different files. Please check and confirm they will not affect GEFS

Only in gsi_github/fix: anavinfo_fv3aerorad
Only in gsi_github/fix: fv3aerorad_satinfo.txt
Only in gsi_vlab/libsrc: GSD
Only in gsi_vlab/util/Ozone_Monitor/image_gen/html: ozn.tar
Only in gsi_vlab/util/Ozone_Monitor/nwprod/gdas_oznmon.v2.0.0/fix: gdas_oznmon_base.tar
Only in gsi_vlab/util/Radiance_Monitor/nwprod/gdas_radmon.v3.0.0/fix: gdas_radmon_base.tar

WalterKolczynski-NOAA commented:
I got an even shorter list:

m71a3 GSI> git branch
   master
* release/gefs_v12
m71a3 GSI> git diff vlab/release/gefs_v12 --
diff --git a/util/Ozone_Monitor/image_gen/html/ozn.tar 
b/util/Ozone_Monitor/image_gen/html/ozn.tar
deleted file mode 100644
index edc9555d3..000000000
Binary files a/util/Ozone_Monitor/image_gen/html/ozn.tar and /dev/null 
differ
diff --git 
a/util/Ozone_Monitor/nwprod/gdas_oznmon.v2.0.0/fix/gdas_oznmon_base.tar 
b/util/Ozone_Monitor/nwprod/gdas_oznmon.v2.0.0/fix/gdas_oznmon_base.tar
deleted file mode 100644
index 0526bd1b1..000000000
Binary files 
a/util/Ozone_Monitor/nwprod/gdas_oznmon.v2.0.0/fix/gdas_oznmon_base.tar 
and /dev/null differ
diff --git 
a/util/Radiance_Monitor/nwprod/gdas_radmon.v3.0.0/fix/gdas_radmon_base.tar 
b/util/Radiance_Monitor/nwprod/gdas_radmon.v3.0.0/fix/gdas_radmon_base.tar
deleted file mode 100644
index 9603dc9d3..000000000
Binary files 
a/util/Radiance_Monitor/nwprod/gdas_radmon.v3.0.0/fix/gdas_radmon_base.tar 
and /dev/null differ

I think you've got other non-repo stuff in there.

On 2020-07-30 16:49, Xianwu Xue - NOAA wrote:
>
> I just compared the gsi_vlab vs gsi_github, and below are the 
> different files. Please check and confirm they will not affect GEFS
>
> Only in gsi_github/fix: anavinfo_fv3aerorad
> Only in gsi_github/fix: fv3aerorad_satinfo.txt
> Only in gsi_vlab/libsrc: GSD
> Only in gsi_vlab/util/Ozone_Monitor/image_gen/html: ozn.tar
> Only in gsi_vlab/util/Ozone_Monitor/nwprod/gdas_oznmon.v2.0.0/fix: 
> gdas_oznmon_base.tar
> Only in gsi_vlab/util/Radiance_Monitor/nwprod/gdas_radmon.v3.0.0/fix: 
> gdas_radmon_base.tar
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/NOAA-EMC/global-workflow/pull/111#issuecomment-666686730>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AGWYBOBWOICAQFZXA3XLDXDR6HMEXANCNFSM4POH3YOA>.
>


-- 
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus
